# задание 1

Как относятся друг к другу множество и последовательность? (в ответе использовать слова типа: часть, целое, общее, частное, родитель, дочерний субъект и т.д.)

ответ: множество это набор определенных чисел без какой либо зависимости можно сказать родитель последовательности, последовательность это дочерний субъект множества в котором числа расположены в определенном порядке(порядок задается формулой)

### задание 2

Даны 4 последовательности. Необходимо:

а) исследовать их на монотонность; 

б) исследовать на ограниченность;

в) найти пятый по счету член.



### 1
$\lbrace a_n\rbrace _{n=1}^{\infty} = 2^n - n$


при помощи пайтон кода найдем первые 5 членов последовательности

In [16]:
[print(f'a{n}:{2**n - n}') for n in range(1,6)]

a1:1
a2:2
a3:5
a4:12
a5:27


[None, None, None, None, None]

последовательность возрастает 

$[0:\infty)$

пятый член = 27

### 2
$\lbrace b_n\rbrace _{n=2}^{\infty} = \frac1{1-n}$

при помощи пайтон кода найдем первые 5 членов последовательности

In [17]:
[print(f'b{n -1}:{1/(1 - n)}') for n in range(2,7)]

b1:-1.0
b2:-0.5
b3:-0.3333333333333333
b4:-0.25
b5:-0.2


[None, None, None, None, None]

последовательность бесконечно стремиться к 0 

$[-1:0)$

пятый член = -0.2

### 3
$\lbrace c_n\rbrace _{n=1}^{\infty} = -1^n + \sqrt{2n}$

In [18]:
[print(f'c{n}:{(-1)**n + (2 * n)**0.5}') for n in range(1,6)]

c1:0.41421356237309515
c2:3.0
c3:1.4494897427831779
c4:3.8284271247461903
c5:2.1622776601683795


[None, None, None, None, None]

последовательность возрастающая 

$[0.41:\infty)$ или $[\sqrt{2} - 1 : \infty)$

пятый член = 2.16 или $\sqrt{10} - 1$

### 4
$\lbrace d_n\rbrace _{n=1}^{\infty} = (-1)^{2n} + \frac1{n^2}$

In [19]:
[print(f'd{n}:{(-1)**(2*n) + 1 / (n**2)}') for n in range(1,6)]

d1:2.0
d2:1.25
d3:1.1111111111111112
d4:1.0625
d5:1.04


[None, None, None, None, None]

последовательность убывает и стремиться к 1

(1:2]

пятый член = 1.04

### задание 3
$a_1 = 128, a_{n+1} - a_n = 6$

исходя из формулы каждый член последовательности будет равен:

$a_{n+1}= 6 - a_n$

теперь при помощи кода пайтон найдем 12 член последовательности

In [22]:
a = 128
for i in range(2,13):
    next_a = 6 - a
    print(f'a{i}:{next_a}')
    a = next_a

a2:-122
a3:128
a4:-122
a5:128
a6:-122
a7:128
a8:-122
a9:128
a10:-122
a11:128
a12:-122


# задание 4

In [52]:
import math
n = 1
old = 0
for i in range(170):
    new = n/(math.factorial(n)**(1/n))
    e = new - old
    old = new
    n = n + 1
    
print(f'new:{new}, old:{old}, e:{e}')
    

new:2.663087878748024, old:2.663087878748024, e:0.0002768007292406338
